olp-cpp-sdk-core/src/client/OlpClient.cpp

```diff
@@ -291,6 +300,18 @@ CancellationToken OlpClient::CallApi(
     const std::shared_ptr<std::vector<unsigned char>>& post_body,
     const std::string& content_type,
     const NetworkAsyncCallback& callback) const {
+  auto http_verb = GetHttpVerb(method);
+  return CallApi(path, http_verb, query_params, header_params, form_params, post_body, content_type, callback);
```

Comment:
```suggestion
  return CallApi(path, http_verb, query_params, header_params, form_params,
                 post_body, content_type, callback);
```
Clang format please. Exceeds max line width